### PR TITLE
Readme and license

### PR DESCRIPTION
**Pull request by CHBMB**
Just because I'm playing around with CodiMD and figured I'd make this pretty.  Checked that having a `.md` file amidst all the proxy confs doesn't cause any issues in our LetsEncrypt image.  Also added a LICENSE.

### CODE REVIEW

**Comment by nemchik** [comment]
This was next on my list of things to propose :smile:

**Comment by CHBMB** [comment]
@nemchik You snooze, you lose :stuck_out_tongue_closed_eyes:

**Comment by nemchik** [comment]
You may want to adjust https://github.com/linuxserver/docker-letsencrypt/blob/master/Dockerfile line 106 to exclude the licence file (no real need for it to be tossed in with the proxy configs).

File: README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+### Make any necessary changes detailed in the config
+
+Some applications require you to make changes to the service containers such as adding base urls in their settings. Each conf file lists the required changes on the first line.
+
+If you are reverse proxying linuxserver containers installed on the same host with the recommended options, you shouldn't need to edit these conf files.
+
+## To disable the configs:
+
+Simply rename the files to add back the `.sample` to the end, and restart letsencrypt.
```
Comment (aptalca) [inline README.md]: @CHBMB 
Let's change this to `Simply delete the confs and restart letsencrypt.`

Comment (CHBMB) [reply README.md]: Done

**Pull request merged**